src/test/fuzz/script.cpp

```diff
@@ -82,7 +89,6 @@ FUZZ_TARGET_INIT(script, initialize_script)
     (void)script.IsPayToScriptHash();
     (void)script.IsPayToWitnessScriptHash();
     (void)script.IsPushOnly();
-    (void)script.IsUnspendable();
```

Comment:
What do these lines do? Push the output to `solutions`?

Reply:
Nothing. They increase coverage.